karma.conf.babel.js

```diff
@@ -0,0 +1,80 @@
+import paths from './paths';
```

Comment:
Looks like we're [using `babel-node`](https://github.com/TechnologyAdvice/tap-web-app/pull/9/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R17) to kick off the test runner. As a result, shouldn't our presets in the `.babelrc` handle the transpilation for us?